project 2/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+   <title>Project 2</title>
+   <link rel="stylesheet" href="styles.css">
+   <link href="https://fonts.googleapis.com/css?family=Big+Shoulders+Display:500,600,800,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section>
+      <div class="content"> 
+      
+      <img src="images/海报 copy.jpg" alt="">
```

Comment:
Also, note there are two issues with your image:
1. You never want a space in a filename that you're going to serve online. You can have hyphens or underscores (so `moon_poster.jpg` is okay but not `moon poster.jpg`).
1. This image hasn't been resized or optimized for the web. It's almost 2000px wide, but the width in your CSS is 124px -- so it's _much much_ larger than it needs to be for your design. Additionally, [tinyPNG](https://tinypng.com/) says it can be optimized for a filesize savings of 76%. As you can imagine, on a site with many movie cards on a single page, those savings would add up, especially for mobile users.